src/test/codegen/fastcall-inreg.rs

```diff
@@ -70,7 +70,7 @@ pub mod tests {
     #[no_mangle]
     pub extern "fastcall" fn f5(_: i64, _: i32) {}
 
-    // CHECK: @f6(i1 inreg zeroext %_1, i32 inreg %_2, i32 %_3)
+    // CHECK: @f6(i1 inreg noundef zeroext %_1, i32 inreg %_2, i32 %_3)
```

Comment:
Could you please change this test to use `--target i386-unknown-linux-gnu` together with `needs-llvm-components: x86`?